src/pages/projectSettings/tabs/ViewsAndFiltersTab/getFormSchema.ts

```diff
@@ -12,7 +14,54 @@ export const getFormSchema = (): ReturnType<GetFormSchema> => ({
       },
       parsleyFilters: {
         title: "Parsley Filters",
-        type: "object" as "object",
+        type: "array" as "array",
+        default: [],
+        items: {
+          type: "object" as "object",
+          properties: {
+            expression: {
+              type: "string" as "string",
+              title: "Filter Expression",
+              minLength: 1,
+              default: "",
+              format: "validRegex",
+            },
+            caseSensitivity: {
+              type: "string" as "string",
+              title: "Case",
+              default: CaseSensitivity.Insensitive,
+              oneOf: [
+                {
+                  type: "string" as "string",
+                  title: "Insensitive",
+                  enum: [CaseSensitivity.Insensitive],
+                },
+                {
+                  type: "string" as "string",
+                  title: "Sensitive",
+                  enum: [CaseSensitivity.Sensitive],
+                },
+              ],
+            },
```

Comment:
An alternate approach here and for matchType could be to use booleans as the `enum` values. This would prevent you from having to convert from bools to strings back to bools in the transformer functions. The enums do have the benefit of being readable but I don't think booleans are too confusing either. Your call!

```suggestion
            caseSensitivity: {
              type: "boolean" as "boolean",
              title: "Case",
              default: false,
              oneOf: [
                {
                  type: "boolean" as "boolean",
                  title: "Insensitive",
                  enum: [false],
                },
                {
                  type: "boolean" as "boolean",
                  title: "Sensitive",
                  enum: [true],
                },
              ],
            },
```